packages/fxa-settings/src/pages/CookiesDisabled/index.test.tsx

```diff
@@ -0,0 +1,136 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import CookiesDisabled, { routeName } from '.';
+import { screen, render, fireEvent } from '@testing-library/react';
+import { logViewEvent, logPageViewEvent } from '../../lib/metrics';
+
+jest.mock('../../lib/metrics', () => ({
+  logPageViewEvent: jest.fn(),
+  logViewEvent: jest.fn(),
+}));
+
+const originalWindow = window;
+function setLocationSearch(search = '') {
+  // @ts-ignore
```

Comment:
I _really_ tried to _not_ use this approach since it needs that `@ts-ignore` but there is still not a good way to mock these values otherwise. This was the cleanest way that worked.

Reply:
I think this works great!